.eslintrc.json

```diff
@@ -20,6 +50,10 @@
         "ignoreDestructuring": true
```

Comment:
Why ignore destructuring?

Reply:
It's only for attributes in camelCase objects, as:
myVar: { someone_made_this: false}

I think that is really really rare and only useful in some third party object/data;

Reply:
But this rule won't turn any destructuring unecessary?